smsdk/_version.py

```diff
@@ -0,0 +1,82 @@
+#
+"""
+Track version information for factorytx-core module here.
+"""
+import typing
+import itertools
+import requests
+import warnings
+import functools
+
+
+class VersionInfo(typing.NamedTuple):
+    major: typing.Any
+    minor: typing.Any
+    patchlevel: typing.Any
+    releaselevel: typing.Any
+    serial: typing.Any
+
+    def __str__(self) -> str:
+        result = "".join(
+            itertools.chain(
+                ".".join(str(i) for i in self[:3] if i is not None),
+                (str(i) for i in self[3:] if i is not None),
+            )
+        )
+        return result
+
+
+version_info = VersionInfo(1, 26, 0, "", None)
+version = str(f"v{version_info}")
+
+
+def get_latest_sdk_release() -> typing.Optional[typing.Any]:
+    api_url = f"https://api.github.com/repos/{owner}/{repo}/releases"
```

Comment:
I prefer parameters to be function parameters than magic globals.  If they are global constants use CAPITALS for their names.

Reply:
Replaced the global variable with a class attribute